src/config/WorkflowList.json

```diff
@@ -1,48 +1,135 @@
-        "suffix": "UPPA",
-        "text": "UPPA - Pre PCR Amplification",
-        "value": "UPPA",
-        "workflow": "PacBio UltraLow"
+        "suffix": "PULPPA",
+        "text": "PULPPA - Pre PCR Amplification",
+        "value": "PULPPA",
+        "workflow": "Pacbio ULI LP"
       },
       {
         "stage": "Post Amplification A",
-        "suffix": "UPAA",
-        "text": "UPAA - Post Amplification A",
-        "value": "UPAA",
-        "workflow": "PacBio UltraLow"
+        "suffix": "PULPAA",
+        "text": "PULPAA - Post Amplification A",
+        "value": "PULPAA",
+        "workflow": "Pacbio ULI LP"
       },
       {
         "stage": "Post Amplification B",
-        "suffix": "UPAB",
-        "text": "UPAB - Post Amplification B",
-        "value": "UPAB",
-        "workflow": "PacBio UltraLow"
+        "suffix": "PULPAB",
+        "text": "PULPAB - Post Amplification B",
+        "value": "PULPAB",
+        "workflow": "Pacbio ULI LP"
       },
       {
         "stage": "Post Pool pre-Size Selection",
-        "suffix": "UPPS",
-        "text": "UPPS - Post Pool pre-Size Selection",
-        "value": "UPPS",
-        "workflow": "PacBio UltraLow"
+        "suffix": "PULPPS",
+        "text": "PULPPS - Post Pool pre-Size Selection",
+        "value": "PULPPS",
+        "workflow": "Pacbio ULI LP"
       },
       {
         "stage": "Final Library",
-        "suffix": "USBT",
-        "text": "USBT - Final Library",
-        "value": "USBT",
-        "workflow": "PacBio UltraLow"
+        "suffix": "PULSBT",
+        "text": "PULSBT - Final Library",
+        "value": "PULSBT",
+        "workflow": "Pacbio ULI LP"
+      }
+    ]
+  },
+  {
+    "workflow": "Pacbio ULI SP",
+    "options": [
+      {
+        "stage": "Sheared + 3.1xSPRI",
+        "suffix": "PUSSHS",
+        "text": "PUSSHS - Sheared + 3.1xSPRI",
+        "value": "PUSSHS",
+        "workflow": "Pacbio ULI SP"
+      },
+      {
+        "stage": " 3.1xSPRI",
+        "suffix": "PUSSPR",
+        "text": "PUSSPR - 3.1xSPRI",
+        "value": "PUSSPR",
+        "workflow": "Pacbio ULI SP"
+      },
+      {
+        "stage": "Sheared",
+        "suffix": "PUSSHR",
+        "text": "PUSSHR - Sheared",
+        "value": "PUSSHR",
+        "workflow": "Pacbio ULI SP"
+      },
+      {
+        "stage": "Excess",
+        "suffix": "PUSEXC",
+        "text": "PUSEXC - Excess",
+        "value": "PUSEXC",
+        "workflow": "Pacbio ULI SP"
+      }
+    ]
+  },
+  {
+    "workflow": "Pacbio Standard",
+    "options": [
+      {
+        "stage": "Supernatant",
+        "suffix": "PSSUP",
+        "text": "PSSUP - Supernatant",
+        "value": "PSSUP",
+        "workflow": "Pacbio Standard"
+      },
+      {
+        "stage": "Sheared",
+        "suffix": "PSSHR",
+        "text": "PSSHR - Sheared",
+        "value": "PSSHR",
+        "workflow": "Pacbio Standard"
+      },
+      {
+        "stage": "Post SPRI",
+        "suffix": "PSSHR",
```

Comment:
In the story, the suffix given is 'PSSPR' for Post SPRI. So is it need to be changed to 'PSSPR'?
![Screenshot 2024-08-15 at 10 04 51](https://github.com/user-attachments/assets/770eb96e-723a-477c-a76d-06933e1c714f)